pkg/dcap/cert.go

```diff
@@ -126,15 +126,15 @@ func parseTCB(sgxExtMap map[string]asn1.RawValue, compSVNOIDs []asn1.ObjectIdent
 	var sequence []Ext
 	_, err := asn1.Unmarshal(pceSVNRaw.FullBytes, &sequence)
 	if err != nil {
-		return tcb, errors.New(fmt.Sprintf("failed to unmarshal ASN.1 sequence: %w", err))
+		return tcb, errors.New(fmt.Sprintf("failed to unmarshal ASN.1 sequence: %v", err))
```

Comment:
Why change to `%v`?

Reply:
Do not need to preserve the context of the error